### ISSUE TITLE
validation loss

### ISSUE THREAD

**Issue by xliucs** [tristandeleu/pytorch-maml#12]
Hi, Thanks for releasing this useful repo. I have extended this with my own dataset.

I found out the validation loss was dropped significantly in the first a few epochs, but the validation loss was not stable and jumping around a lot since then. Just wondering what is the potential reason for that or has anyone experienced this before? Thanks!
